Project1/main.asm

```diff
@@ -109,6 +110,53 @@ delay1:
 	brne loop1
 	ret
 
+jingleFeature:
+	lds r0, buttonJustPressed
+	lds r16, toneGenFreq
+
+	ldi r21, 0b00010000 ; registers are loaded with binary values to trigger lights for jingle
+	ldi r22, 0b00100000
+	ldi r23, 0b01000000
+
+	sbrc r0, 5 ; play jingle if button 5 is pressed
```

Comment:
This only skips one instruction. The jingle is played without pressing the button always.